src/app/store.ts

```diff
@@ -1,6 +1,13 @@
-import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import { reducer as filterReducer } from '../features/filter';
+import todosReducer from '../features/todos';
+import curentTodoReducer from '../features/currentTodo';
```

Comment:
There is a typo in the import statement for `curentTodoReducer`. It should be `currentTodoReducer` to match the expected naming convention.